packages/devextreme/js/__internal/ui/chat/chat_avatar.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
We shouldn't add this ignore for all file. It is necessary to eliminate the use of any. In extreme cases, it is necessary to ignore the error only for a specific line

Reply:
[90dd3acd3c6dd8b06c21608a237d774ee8b6aa43]